### DISCUSSION TITLE
old slidercircle stick to beginning

### DISCUSSION THREAD

**Discussion by Tash2314** [ppy/osu#28425] [Ideas]
the idea here is to bring a new option to the skin.ini that makes it possible to activate that the sliderhitcircle is sticking to the beginning of the slider like it was by default in the old days of osu 

preview how it looked like: https://www.youtube.com/watch?v=zrCM9hxdM1s&t=58s

**Comment by Tash2314** [comment]
idk if its even possible bc the game evolved and it got putten out for a reason but would be still nice to have the option

**Comment by bdach** [comment]
If you mean slider snaking then that's already in the game. Search "slider snaking" in game options (and turn it off).

**Reply by Tash2314** [reply]
No if you look in the video the cycle at the beginning on the slider that stays there after hitting (can not be made by skin or by changing options bc its got deleated)

**Reply by bdach** [reply]
No idea what this means sorry. You'll have to try and explain better because I don't understand what I'm supposed to be seeing.

**Reply by Stoppedpuma** [reply]
Pretty sure they mean keeping the slider head intact as shown in these pictures: 
![image](https://github.com/ppy/osu/assets/58333920/fcfac92b-5418-4f76-ab1b-398c4ed8efb5)
![image](https://github.com/ppy/osu/assets/58333920/8d60c826-9a88-40c3-b475-5c72676c2aeb)
